src/main/perl/RuleBasedEditor.pm

```diff
@@ -0,0 +1,1023 @@
+# ${license-info}
+# ${developer-info}
+# ${author-info}
+# ${build-info}
+#
+#
+# This module implements a rule-based editor that is used to modify the content
```

Comment:
ah, if this is the same text as uunder the `head1 DESCRIPTION`, please remove it here